eviloauth/dispatcher.py

```diff
@@ -1,86 +1,297 @@
 import sys
 import logging
+from diskcache import Cache
 from eviloauth.idp import IDP
 from eviloauth.exceptions import EviloauthCommandException
+from eviloauth.access_token import AccessToken
+from eviloauth import app, cache, temp_dir, load_modules
+from datetime import datetime, timedelta
 
+class TokenManager:
+    def __init__(self, cache_dir, general_cache):
+        self.cache = Cache(cache_dir)
+        self.general_cache = general_cache
 
-class Dispatcher:
-    def __init__(self, flask_server, module_dict, cache, redirect_server):
-        logging.debug('Initializing dispatcher')
-        logging.debug(f'\tFlask server: {flask_server}')
-        logging.debug(f'\tModule dict: {module_dict}')
```

Comment:
I don't know why you've removed the debugging stuff. It's here to help debugging...

Add it back in